views/machine_detail_widget1.php

```diff
@@ -1,7 +1,7 @@
 <div class="col-lg-4">
     <div class="row">
         <div class="col-xs-6">
-            <img id="apple_hardware_icon" class="img-responsive">
```

Comment:
This will break the ibridge module. References to this should stay as `apple_hardware_icon`